spec/defines/cron_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe 'logrotate::cron' do
+  _, facts = on_supported_os.first
```

Comment:
why .first instead of iterating on all platforms?

Reply:
I had used one of the existing spec files as a template for this. I'll improve the test set to cover all supported os.